README.md

```diff
@@ -27,6 +27,7 @@ npm install pino-elasticsearch -g
   -t  | --type              the name of the type to use; default: log
   -b  | --size              the number of documents for each bulk insert
   -l  | --trace-level       trace level for the elasticsearch client, default 'error' (info, debug, trace).
+  -c  | --aws-credentials   path to aws_config.json (is using AWS Elasticsearch)
```

Comment:
can you add a quick example on how to use this with AWS?

it should be "if using AWS Elasticsearch".